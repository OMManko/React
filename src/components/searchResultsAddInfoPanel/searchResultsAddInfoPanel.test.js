import { shallow } from 'enzyme';
import React from 'react';
import SearchResultsAddInfoPanel from './searchResultsAddInfoPanel';

describe('<SearchResultsAddInfoPanel />', () => {
    it('should render search results additional info panel and match snapshot', () => {
        const component = shallow(<SearchResultsAddInfoPanel/>);
        expect(component).toMatchSnapshot();
    });
});
