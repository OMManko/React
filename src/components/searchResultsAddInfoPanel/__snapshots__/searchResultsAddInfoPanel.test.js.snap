// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchResultsAddInfoPanel /> should render search results additional info panel and match snapshot 1`] = `ShallowWrapper {}`;
